packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -26,16 +27,20 @@ async function importJSONFile(browser: CompassBrowser, jsonPath: string) {
   const importModal = await browser.$(Selectors.ImportModal);
   await importModal.waitForDisplayed();
 
-  // Confirm
+  // Confirm import.
   await browser.clickVisible(Selectors.ImportConfirm);
 
-  // wait for the done button to appear and then click it
-  const doneButton = await browser.$(Selectors.ImportDone);
-  await doneButton.waitForDisplayed();
-  await browser.clickVisible(Selectors.ImportDone);
+  // Wait for the modal to go away.
+  await importModal.waitForDisplayed({ reverse: true });
 
-  // wait for the modal to go away
-  await importModal.waitForDisplayed({ reverse: false });
+  // Wait for the done toast to appear and close it.
+  const toast = Selectors.ImportSucceededToast;
```

Comment:
Feel free to leave this, but in other tests if we need to refer to the same element like that we do 

```suggestion
  const toastElement = await browser.$(Selectors.ImportSucceededToast);
```

instead of copying a string in another variable